homeassistant/components/roborock/strings.json

```diff
@@ -455,6 +455,20 @@
     "get_vacuum_current_position": {
       "name": "Get current position",
       "description": "Get the current position of the vacuum."
+    },
+    "clean_rooms": {
+      "name": "Clean rooms",
+      "description": "Clean specific rooms.",
+      "fields": {
+        "room_ids": {
+          "name": "Room IDs",
+          "description": "Ids of the Rooms to clean, separated by comma. (retrieve the ids with the service get_maps)"
```

Comment:
This has to properly explain both id's and names can be used (are they case sensitive?)